src/neo/SmartContract/Native/Oracle/OraclePolicyContract.cs

```diff
@@ -0,0 +1,235 @@
+        /// <returns>Oracle multisignature address</returns>
+        public UInt160 GetOracleMultiSigAddress(StoreView snapshot)
+        {
+            ECPoint[] validators = GetOracleValidators(snapshot);
+            return Contract.CreateMultiSigRedeemScript(validators.Length - (validators.Length - 1) / 3, validators).ToScriptHash();
+        }
+
+        /// <summary>
+        /// Set HttpConfig
+        /// </summary>
+        /// <param name="engine">VM</param>
+        /// <param name="args">Parameter Array</param>
+        /// <returns>Returns true if the execution is successful, otherwise returns false</returns>
+        [ContractMethod(0_03000000, ContractParameterType.Boolean, ParameterTypes = new[] { ContractParameterType.Integer }, ParameterNames = new[] { "httpConfig" })]
+        private StackItem SetHttpConfig(ApplicationEngine engine, Array args)
+        {
+            StoreView snapshot = engine.Snapshot;
+            UInt160 account = GetOracleMultiSigAddress(snapshot);
+            if (!InteropService.Runtime.CheckWitnessInternal(engine, account)) return false;
+            int timeOutMilliSeconds = (int)args[0].GetBigInteger();
+            if (timeOutMilliSeconds <= 0) return false;
+            OracleHttpConfig httpConfig = new OracleHttpConfig() { Timeout = timeOutMilliSeconds };
+            StorageItem storage = snapshot.Storages.GetAndChange(CreateStorageKey(Prefix_HttpConfig));
+            storage.Value = httpConfig.ToArray();
+            return true;
+        }
+
+        /// <summary>
+        /// Get HttpConfig
+        /// </summary>
+        /// <param name="engine">VM</param>
+        /// <returns>value</returns>
+        [ContractMethod(0_01000000, ContractParameterType.Array)]
+        private StackItem GetHttpConfig(ApplicationEngine engine, Array args)
+        {
+            return GetHttpConfig(engine.Snapshot).ToStackItem(engine.ReferenceCounter);
+        }
+
+        /// <summary>
+        /// Get HttpConfig
+        /// </summary>
+        /// <param name="snapshot">snapshot</param>
+        /// <returns>value</returns>
+        public OracleHttpConfig GetHttpConfig(StoreView snapshot)
+        {
+            return snapshot.Storages[CreateStorageKey(Prefix_HttpConfig)].Value.AsSerializable<OracleHttpConfig>();
+        }
+
+        /// <summary>
+        /// Set PerRequestFee
+        /// </summary>
+        /// <param name="engine">VM</param>
+        /// <param name="args">Parameter Array</param>
+        /// <returns>Returns true if the execution is successful, otherwise returns false</returns>
+        [ContractMethod(0_03000000, ContractParameterType.Boolean, ParameterTypes = new[] { ContractParameterType.Integer }, ParameterNames = new[] { "fee" })]
+        private StackItem SetPerRequestFee(ApplicationEngine engine, Array args)
+        {
+            StoreView snapshot = engine.Snapshot;
+            UInt160 account = GetOracleMultiSigAddress(snapshot);
```

Comment:
Should be configured by CN or by Oracles?

Reply:
I prefer CN, as incentive will be simple.

Reply:
Then should be changed to get the Validators from NEO Native contract

Reply:
Got it

Reply:
If we decide to use the governance mechanism from #1446, it should be configured by the committee.

Reply:
Agree.  
Could we merge first, since we change to `oracle` branch to fast our progress and we can update here after governance mechanism decided?

Reply:
> Could we merge first, since we change to oracle branch to fast our progress and we can up
date here after governance mechanism decided?

It's ok for me